### DISCUSSION TITLE
Add `facebook_id` and `youtube_id` fields to the `channels.csv`

### DISCUSSION THREAD

**Discussion by freearhey** [iptv-org/community#530] [Polls]
This should make it easier to sort channels especially when they don't have an official website.

**Comment by BellezaEmporium** [comment]
No idea, as I'm quite unsure of why it would be necessary to get social networks out for TV channels. I know though that certain ones rely on FB or Twitter to show news about their feeds or channel, news in general.

**Reply by freearhey** [reply]
I thought we could use their social media pages primarily to keep their other data up to date: logos, names, website URLs, etc.
